langchain4j-core/src/main/java/dev/langchain4j/agent/tool/ToolJsonSchema.java

```diff
@@ -0,0 +1,52 @@
+package dev.langchain4j.agent.tool;
+
+import dev.langchain4j.jsonschema.JsonSchema;
+
+import java.util.Map;
+
+/**
+ * The JsonSchema that describes a {@link Tool}.
+ */
+public class ToolJsonSchema {
```

Comment:
Is there a reason a new type is introduced instead of using `ToolSpecification`?